docs/content/preview/releases/ybdb-releases/v2.23.md

```diff
@@ -0,0 +1,426 @@
+---
+title: What's new in the v2.23 preview release series
+headerTitle: What's new in the v2.23 preview release series
+linkTitle: v2.23 series (Preview)
+description: Enhancements, changes, and resolved issues in the v2.23 preview release series.
+image: /images/section_icons/quick_start/install.png
+menu:
+  preview_releases:
+    identifier: v2.23
+    parent: ybdb-releases
+    weight: 2799
+type: docs
+---
+
+What follows are the release notes for the YugabyteDB v2.23 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v2.23 release series.
+
+For an RSS feed of all release series, point your feed reader to the [RSS feed for releases](../../index.xml).
+
+## v2.23.0.0 - September 6, 2024 {#v2.23.0.0}
+
+**Build:** `2.23.0.0-b710`
+
+### Downloads
+
+<ul class="nav yb-pills">
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.23.0.0/yugabyte-2.23.0.0-b710-darwin-x86_64.tar.gz">
+      <i class="fa-brands fa-apple"></i>
+      <span>macOS</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.23.0.0/yugabyte-2.23.0.0-b710-linux-x86_64.tar.gz">
+      <i class="fa-brands fa-linux"></i>
+      <span>Linux x86</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.23.0.0/yugabyte-2.23.0.0-b710-el8-aarch64.tar.gz">
+      <i class="fa-brands fa-linux"></i>
+      <span>Linux ARM</span>
+    </a>
+  </li>
+</ul>
+
+**Docker:**
+
+```sh
+docker pull yugabytedb/yugabyte:2.23.0.0-b710
+```
+
+### Highlights
+
+We're excited to announce the [technical preview](/preview/releases/versioning/#tech-preview-tp) of four powerful new features in YugabyteDB aimed at simplifying operations and enhancing functionality.
+
+**Database instant cloning** {{<badge/tp>}}
+
+[Database Cloning](/preview/manage/backup-restore/lightweight-db-clone/) allows you to quickly create independent copies of your database for data recovery, development, and testing. Clones can be created in seconds, and initially consume no additional disk space, because they share the same data files as the original database at creation but operate independently thereafter. Clones can be created as of now, or as of any time in the recent past (within a configurable history retention period), enabling developers to experiment without impacting production performance. This feature also provides a safety net for rapid recovery from accidental data loss or corruption.
```

Comment:
```suggestion
[Database Cloning](/preview/manage/backup-restore/lightweight-db-clone/) allows you to quickly create independent copies of your database for data recovery, development, and testing. Clones can be created in seconds, and initially consume no additional disk space because they share the same data files as the original database. Clones can be created as of now, or as of any time in the recent past (within a configurable history retention period), so developers can experiment without impacting production performance. Clones also provide a safety net for rapid recovery from accidental data loss or corruption.
```